views/js/qtiCreator/test/MathEditor/test.js

```diff
@@ -0,0 +1,67 @@
+define([
```

Comment:
FYI, all tests are supposed to be located under `views/js/test` to be executed by the CI engine. This folder is an old one and will be migrated at some point.
Could you please make the test fails if MathJax is not present, so when we migrate this folder we now why the test do not run.